src/redux/sagas/saga.js

```diff
@@ -32,10 +32,10 @@ function* watchAddAlbum(){
 
 
 
-function* fetchImages(ID) {
+function* fetchImages(albumID) {
```

Comment:
As usual var names should be camel case e.g. albumId

Reply:
Changed it everywhere.